apps/hash-frontend/src/pages/[shortname].page/profile-page-header.tsx

```diff
@@ -70,42 +89,86 @@ export const ProfilePageHeader: FunctionComponent<{
               }}
             />
           )}
-          <Box paddingTop={`${avatarTopOffset}px`}>
-            {profile ? (
-              <Typography
-                variant="h1"
-                sx={{
-                  fontSize: 32,
-                  fontWeight: 700,
-                  color: ({ palette }) => palette.gray[90],
-                }}
-              >
-                {profile.kind === "user" ? profile.preferredName : profile.name}
-              </Typography>
-            ) : (
-              <Skeleton
-                height={50}
-                width={200}
-                sx={{ background: ({ palette }) => palette.gray[20] }}
-              />
-            )}
-            {profile ? (
-              <Typography
-                sx={{
-                  fontSize: 14,
-                  fontWeight: 500,
-                  color: ({ palette }) => palette.gray[70],
-                }}
-              >
-                @{profile.shortname}
-              </Typography>
-            ) : (
-              <Skeleton
-                height={30}
-                width={150}
-                sx={{ background: ({ palette }) => palette.gray[20] }}
+          <Box
+            paddingTop={`${avatarTopOffset}px`}
+            display="flex"
+            flexDirection="column"
+            justifyContent="space-between"
+            flexGrow={1}
+          >
+            <Box>
+              {profile ? (
+                <Typography
+                  variant="h1"
+                  sx={{
+                    fontSize: 32,
+                    fontWeight: 700,
+                    color: ({ palette }) => palette.gray[90],
+                  }}
+                >
+                  {profile.kind === "user"
+                    ? profile.preferredName
+                    : profile.name}
+                </Typography>
+              ) : (
+                <Skeleton
+                  height={50}
+                  width={200}
+                  sx={{ background: ({ palette }) => palette.gray[20] }}
+                />
+              )}
+              {profile ? (
+                <Typography
+                  sx={{
+                    fontSize: 14,
+                    fontWeight: 500,
+                    color: ({ palette }) => palette.gray[70],
+                  }}
+                >
+                  @{profile.shortname}
+                </Typography>
+              ) : (
+                <Skeleton
+                  height={30}
+                  width={150}
+                  sx={{ background: ({ palette }) => palette.gray[20] }}
+                />
+              )}
+            </Box>
+            <Box display="flex" justifyContent="space-between">
+              <ProfilePageTabs
+                profile={profile}
+                tabs={tabs}
+                currentTab={currentTab}
               />
-            )}
+              {isEditable ? (
+                <>
+                  <Button
```

Comment:
I would remove the background and border on hover as we don't do this for other similar links... I think the text changing colour is sufficient
![Screenshot 2023-10-10 at 12 23 59](https://github.com/hashintel/hash/assets/37743469/e16934ce-bddb-408d-b913-5965894b0df3)

Reply:
This button is going to be removed once I implement [the updated designs](https://www.figma.com/file/gydVGka9FjNEg9E2STwhi2/HASH?node-id=9717%3A50623&mode=dev) as part of https://linear.app/hash/issue/H-931/display-cover-image-in-user-profile-page, so I think I'll just leave it as is for now.

Reply:
Fair enough!